### DISCUSSION TITLE
Broken/deactivated Server?

### DISCUSSION THREAD

**Discussion by m-topor** [mastodon/mastodon#30229] [Q&A]
Hey, been searching for an answer for some time and I have no idea where else to ask. 

I had an account on https://neuroscience-mastodon.com/web 
I cannot access the server, the page does not open at all. I can't log in to my account and I fear that I had lost access permanently if the server is deactivated. 

What can I do?

**Comment by StrangeMinds1960** [comment]
Showing a 502 / Bad gateway error for me.  Hopefully the site owner knows about it and is attempting a fix

**Comment by m-topor** [comment]
Is it possible to identify the site owner and get in touch directly?

**Reply by StrangeMinds1960** [reply]
Site can't be found on the few Mastodon listing sites I know about, and the whois data for the domain is private

Not sure what to try next

**Reply by m-topor** [reply]
I suspect the server is abandoned. Any way to access abandoned servers or is access lost with the owner?

**Reply by p37307** [reply]
You can try to contact them through their official means of domain communications. No guarantee they will email you back. You can generate a contact email for them from Google, their registrar, at https://domains.google.com/contactregistrant/?domain=neuroscience-mastodon.com&pli=1
